docs/getting-started/storing-data.mdx

```diff
@@ -199,12 +203,12 @@ Follow along in your `contracts/incrementor/src/lib.rs` file.
 
 ### Contract Data Keys
 
-Contract data is associated with a key. The key is the value that can be used at
+Contract data is associated with a key. The key can be used at
 a later time to lookup the value.
```

Comment:
While we're here we may as well address the "'lookup' is a noun; 'look up' is a verb" issue

```suggestion
a later time to look up the value.
```

(similar to "login" vs "log in" or "setup" vs "set up")